EMS/admin-ui-bundle/assets/css/core/components/media_library.scss

```diff
@@ -73,7 +73,7 @@ div.media-lib {
       display: flex;
       flex-wrap: wrap;
       border-bottom: 1px solid #DCDCDC;
-      gap: 8px 8px;
```

Comment:
Did you csslint the code? How did you find this ?

Reply:
csslint :) after npm update